relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -1012,6 +1014,17 @@ mod tests {
                     "status": "ok",
                     "data": {}
                 },
+                {
+                    "description": "SET 'aaa:bbb:123:zzz' '{\"from json\": \"no\"}'",
```

Comment:
Without the change above, Relay would extract `span.action=json` from this description.

Reply:
So that means the regex is quite wrong?
I would expect it to extract `SET` or what kind of `action` would it be from the product point of view?

Reply:
Oops, I meant `domain`. Updated the PR title, description, and message above to reflect it.